docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_JDBC_Store_for_Messaging_Journal.adoc

```diff
@@ -50,6 +33,19 @@ The name of these tables can be configured with the
 `journal-bindings-table`, `journal-messages-table`,
 `journal-page-store-table, and journal-large-messages-table`.
 
+Please note that the configuration of the underlying pool is something that you need to take care of.
+The you  need at least four connections:
+
+ * one for the binding
+ * one for the messages journal
+ * one for the lease lock (if you use HA)
+ * one for the node manager shared state (if you use HA)
+
+But one of the fact that you need to take into account is that paging and large messages can use an unbounded number of connections.
+The size of the pool should be defined according to the amount of concurrent threads that perform page/large message streaming operations.
```

Comment:
> The size of the pool should be defined

AFAIU the size is dynamic and can't be hard-coded in some configuration. But user could define is max-size of the pool. Right?

Reply:
> according to the amount of concurrent threads

Can these amount of threads be read somewhere? (eg `/subsystem=ee/managed-thread-factory=...:read-resource()`?) Can you please specify some rules for the size of pool based on the amount of concurrent threads?

Reply:
Correct, you can configure a min size and a max size. Also the timeout to get a connection, or the time to return the connection to the pool or renew it. It seems to me that this is more a datasource configuration documentation than one that is proper to the journal

Reply:
Sure, details about datasource commands should be mentioned in datasource chaptes in upstream doc. But so far `The size of the pool should be defined` text in this messaging guides seems to be wrong based on your previous message. Also rules based on messaging streaming operations are still missing here (see https://github.com/wildfly/wildfly/pull/13965/#discussion_r572023909 )